src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/help/Disable-AzureRmRSBRPMountScript.md

```diff
@@ -0,0 +1,135 @@
+---
+external help file: Microsoft.Azure.Commands.RecoveryServices.Backup.dll-Help.xml
+online version: 
+schema: 2.0.0
+---
+
+# Disable-AzureRmRSBRPMountScript
+
+## SYNOPSIS
+Disables access to all the files of the recovery point.
+
+## SYNTAX
+
+```
+Disable-AzureRmRSBRPMountScript [-RecoveryPoint] <RecoveryPointBase> [-PassThru]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Disable-AzureRmRSBRPMountScript** cmdlet disables access to the files of the recovery point which were earlier mounted.
+
+## EXAMPLES
+
+### Example 1: Select the recovery point and get the script
+```
+PS C:\> $namedContainer = Get-AzureRmRecoveryServicesBackupContainer  -ContainerType "AzureVM" �Status "Registered" -FriendlyName "V2VM"
+PS C:\> $backupitem = Get-AzureRmRecoveryServicesBackupItem �Container $namedContainer  �WorkloadType "AzureVM"
```

Comment:
@dragonfly91 it looks like there might have been some bad encoding on the first two lines of this example